app/src/main/java/com/example/placefinder/service/ParkService.kt

```diff
@@ -8,8 +8,12 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
-class ParkService {
-    suspend fun fetchParks(): List<Park>? {
+interface IParkService {
+    suspend fun fetchParks(): List<Park>?
+}
+
+class ParkService : IParkService {
+    override suspend fun fetchParks(): List<Park>? {
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.